### PR TITLE
PH-276: Update Composer dependencies

### PR DESCRIPTION
**Pull request by jmleroux**
Remove a deprecated exception.
And update Composer dependencies by the way.

### CODE REVIEW

File: src/Akeneo/Tool/Bundle/MessengerBundle/Command/PurgeMessengerCommand.php [hunk 1]
```diff
@@ -19,14 +19,11 @@
 class PurgeMessengerCommand extends Command
 {
     protected static $defaultName = 'akeneo:messenger:doctrine:purge-messages';
+    protected static $defaultDescription = 'Purges the messenger SQL table in terms of the given retention time (default is 7200 seconds)';
```
Comment (jmleroux) [inline src/Akeneo/Tool/Bundle/MessengerBundle/Command/PurgeMessengerCommand.php:22]: Lazy command description.
See https://symfony.com/blog/new-in-symfony-5-3-lazy-command-description

File: src/Akeneo/Tool/Bundle/MessengerBundle/Command/PurgeMessengerCommand.php [hunk 2]
```diff
@@ -66,13 +60,13 @@ protected function execute(InputInterface $input, OutputInterface $output): ?int
 
         try {
             $this->purgeDoctrineQueue->execute($tableName, $queueName, $olderThan);
-        } catch (DBALException $dbalException) {
```
Comment (jmleroux) [inline src/Akeneo/Tool/Bundle/MessengerBundle/Command/PurgeMessengerCommand.php:69]: DbalException is deprecated

**Review by BitOne — APPROVED** [review]

**Comment by BitOne** [comment]
Thanks for the cleanup ;)

File: src/Akeneo/Tool/Bundle/MessengerBundle/Command/PurgeMessengerCommand.php [hunk 3]
```diff
@@ -66,13 +60,13 @@ protected function execute(InputInterface $input, OutputInterface $output): ?int
...
-            $output->writeln($dbalException->getMessage());
+        } catch (\Exception $exception) {
```
Comment (StevenVAIDIE) [inline src/Akeneo/Tool/Bundle/MessengerBundle/Command/PurgeMessengerCommand.php:63]: Replacement is \Doctrine\DBAL\Exception but I am ok to catch all exception here

**Review by StevenVAIDIE — APPROVED** [review]

**Pull request merged**